### PR TITLE
2783 bug

### PR DESCRIPTION
**Pull request by Fesmofet**

### CODE REVIEW

File: utilities/helpers/updateSpecificFieldsHelper.js [hunk 1]
```diff
@@ -135,6 +130,95 @@ const update = async (author, permlink, authorPermlink, voter) => {
...
+  const { app: { admins } } = await App.getOne({ name: config.app });
+  const { wobjects: [{ fields } = {}] } = await Wobj.getSomeFields('parent', authorPermlink);
+  // check for admin votes
+  for (const field of fields) {
+    for (const vote of field.active_votes) {
+      if (_.includes(admins, vote.voter)) {
+        return handleAdminVoteOnParent({ authorPermlink, fields, admins });
+      }
+    }
+  }
```
Comment (olegtsigulov) [inline utilities/helpers/updateSpecificFieldsHelper.js:148]: what if there is more than one admin like on 1 field?

File: utilities/helpers/updateSpecificFieldsHelper.js [hunk 2]
```diff
@@ -135,6 +130,95 @@ const update = async (author, permlink, authorPermlink, voter) => {
...
+}) => {
+  const adminActions = [];
+  const noAdminAction = [];
+  // here we find out which fields the admins interacted with
+  for (const field of fields) {
+    const votedField = _.find(field.active_votes, (f) => _.includes(admins, f.voter));
+    if (votedField) {
+      votedField.timestamp = new Date(
+        parseInt(votedField._id.toString().substring(0, 8), 16) * 1000,
+      ).valueOf();
```
Comment (olegtsigulov) [inline utilities/helpers/updateSpecificFieldsHelper.js:167]: votedField._id.getTimestamp().valueOf()

File: utilities/helpers/updateSpecificFieldsHelper.js [hunk 3]
```diff
@@ -135,6 +130,95 @@ const update = async (author, permlink, authorPermlink, voter) => {
...
+      // if admin approve wasn't reject later
+      if (!hasRejectLater.length) {
+        await updateWobjParent({ authorPermlink, parent: approved[0].parent });
+      }
+      // if has not rejected approve
+      if (hasRejectLater.length && notRejected.length) {
+        await updateWobjParent({ authorPermlink, parent: notRejected[0].parent });
+      }
+      // if all fields are rejected by admins
+      if (hasRejectLater && !notRejected.length && (!noAdminAction.length || _.get(noAdminAction, '[0].weight') < 0)) {
```
Comment (olegtsigulov) [inline utilities/helpers/updateSpecificFieldsHelper.js:211]: too match code here

File: utilities/helpers/updateSpecificFieldsHelper.js (hunk 1 above)
Comment (Fesmofet) [reply utilities/helpers/updateSpecificFieldsHelper.js]: +

File: utilities/helpers/updateSpecificFieldsHelper.js (hunk 2 above)
Comment (Fesmofet) [reply utilities/helpers/updateSpecificFieldsHelper.js]: +

File: utilities/helpers/updateSpecificFieldsHelper.js (hunk 3 above)
Comment (Fesmofet) [reply utilities/helpers/updateSpecificFieldsHelper.js]: +

**Pull request merged**